Sources/Jobs/Scheduler.swift

```diff
@@ -97,7 +118,7 @@ private extension Scheduler {
     func checkPropertyWrappers<T: Job>(_ job: T) throws {
         for property in Mirror(reflecting: job).children {
             switch property.value {
-            case is Environment<EnvironmentValues, Connection>:
+            case is Environment<EnvironmentValues, Connection>, is RequestBasedPropertyWrapper:
```

Comment:
Please always use execute of Traversable.swift instead of mirror. Otherwise DynamicProperty and Properties are not supported.

Reply:
I use the methods `apply` and `execute` of `Traversable` now. Since `Job`s have their own target I made both methods public. Furthermore, I declared `Activatable` public to work with `Job`s. This means that `@State` can now be used with `Job`s and is activated only once at creation and stays activated. 

Also, `apply` and `Activatable` need to be public to function with the new approach of `@Environment` #181 which will also conform to `Activatable`.